pkgs/by-name/en/envoy/package.nix

```diff
@@ -239,6 +248,7 @@ buildBazelPackage rec {
     "--linkopt=-Wl,-z,noexecstack"
     "--config=gcc"
     "--verbose_failures"
+    "--incompatible_enable_cc_toolchain_resolution=true"
```

Comment:
@lukegb just me trying to learn here:

So it's basically this one-line change (and line 105 using rustc.unwrapped as the stdlib) that was the "solution" to my post 😆 ?

And `--incompatible_enable_cc_toolchain_resolution=true` is a bazel flag updated in https://github.com/bazelbuild/bazel/issues/7260 which helps discover the proper C++ toolchain to use?

Not sure why this would fix wasmtime which is rust? So maybe line 105 is the fix for wasmtime stuff and this flag is necessary for building Envoy with the wasm support?